sycl/include/CL/sycl/builtins.hpp

```diff
@@ -1632,6 +1632,7 @@ extern SYCL_EXTERNAL double ldexp(double x, int exp);
 extern SYCL_EXTERNAL double hypot(double x, double y);
 }
 #ifdef __GLIBC__
+extern SYCL_EXTERNAL void __failed_assertion();
```

Comment:
I understand all of the changes except this one. Why the new, undefined function declaration?

Reply:
I missed this, it should be removed.
Thanks very much.